src/screens/OrganizationPeople/AddMember.tsx

```diff
@@ -97,7 +97,20 @@ function AddMember(): JSX.Element {
     setCreateNewUserModalIsOpen(!addUserModalisOpen);
 
   const [addMember] = useMutation(ADD_MEMBER_MUTATION);
+  function useModal(initialState = false) {
```

Comment:
1. Make this file use CSS exclusively from `src/style/app.module.css`, with the exception of package based CSS.
1. Delete the CSS file in the directory of this file. Make the file exclusively use CSS from `src/style/app.module.css`
1. Use no CSS in `assets/css/app.css`
1. This will help with future attempts to having Dark Mode
1. Use a variable from `src/style/app.module.css` .
1. Use a variable name that describes the function of the color and not the color itself. For example `--search-button-bg` is much better than `--light-blue` 

- https://github.com/PalisadoesFoundation/talawa-admin/issues/3296